examples/consistency_distillation/train_lcm_distill_lora_sdxl_wds.py

```diff
@@ -657,6 +671,24 @@ def parse_args():
         help="Proportion of image prompts to be replaced with empty strings. Defaults to 0 (no prompt replacement).",
     )
     # ----Latent Consistency Distillation (LCD) Specific Arguments----
+    parser.add_argument(
+        "--use_target_model",
+        action="store_true",
```

Comment:
Should this default to false so existing users are not surprised?

Reply:
The target model will be used only if the `--use_target_model` flag is specified (so existing script calls should work as before).

Reply:
Thanks. My bad.